OrbitQt/main.cpp

```diff
@@ -139,7 +158,12 @@ static void StyleOrbit(QApplication& app) {
       "white; }");
 }
 
-static OrbitQt::DeploymentConfiguration FigureOutDeploymentConfiguration() {
+static std::optional<OrbitQt::DeploymentConfiguration>
+FigureOutDeploymentConfiguration() {
+  if (absl::GetFlag(FLAGS_local)) {
+    return std::nullopt;
```

Comment:
This code is all new to me so bear with me: Couldn't this be handled with another `Deployment` type similar to `OrbitQt::NoDeployment` instead of using `optional`?

Reply:
I thought about that. But decided against because DeploymentConfiguration is something the ServiceDeploymentManager handles. And since we are bypassing the manager completely in this code path I thought this is better reflected by having the optional. Still you're right, it feels a bit hacky.

Reply:
I see, well, it was just a thought, I'll leave you with it! Other than that, looks good!

Reply:
No worries. It was a good thought. You immediately spotted the biggest pain point in here. ;-) Thanks for looking at it.